news/2018-11-18-project-loved-week-of-november-18th.md

```diff
@@ -0,0 +1,217 @@
+
+[![](/wiki/shared/news/2018-11-18-project-loved-week-of-november-18th/osu/senpai-notice-me.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)
+
+[Camellia feat. Nanahira - senpai, notice me!](https://osu.ppy.sh/beatmapsets/843685#osu) by [\_RyuK](https://osu.ppy.sh/users/6304246)  
+*written by [waywern2012](https://osu.ppy.sh/users/5870453)*
+
+It has been proven many times that top players can make amazing maps, and this is no exception. \_RyuK is offering us a map of Camellia's "senpai, notice me!", and it includes cute slider shapes, some technical patterns, frequent slider velocity changes and a banger of a song! It is safe to say that the loved captains definitely noticed this map, as the fastest improving player has made one of the fastest favourite-gaining maps, with over 180 favourites during these three months!
+
+[![](/wiki/shared/news/2018-11-18-project-loved-week-of-november-18th/osu/spica.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)
+
+[Endorfin. - Spica](https://osu.ppy.sh/beatmapsets/575464#osu) by [Settia](https://osu.ppy.sh/users/2445897)  
+*written by [waywern2012](https://osu.ppy.sh/users/5870453)*
+
+"Endorfin. - Spica" is a map with an interesting concept by Settia. There are two ways to play this beatmap, with and without vocals. How? Just enable or disable hitsounds to turn vocals on or off. Packing in 100 favourites, this beatmap shows its potential and is fun to play with or without vocals! Try it yourself!
+
+## <a name="osutaiko" id="osutaiko"></a>osu!taiko
+
+osu!taiko Loved candidates were chosen by [-[ ix Ishida xi ]-](https://osu.ppy.sh/users/242910), [-Kazu-](https://osu.ppy.sh/users/920861), [Backfire](https://osu.ppy.sh/users/263110), [iceOC](https://osu.ppy.sh/users/5482401) and [nyanmi-1828](https://osu.ppy.sh/users/6866480)!
+
+---
+
+[![](/wiki/shared/news/2018-11-18-project-loved-week-of-november-18th/taiko/magus-night-fever-inst.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)
+
+[UNDEAD CORPORATION - Magus night fever inst.](https://osu.ppy.sh/beatmapsets/224810#taiko) by [Ekoro](https://osu.ppy.sh/users/284905)  
+*written by [-Kazu-](https://osu.ppy.sh/users/920861)*
+
+Coming as part of his well-known Sorcery Light series, Ekoro's "Magus night fever inst." is probably one of the hardest high-BPM stamina maps out there. With large streams, both complex and monocolor, this map focuses on emphasizing the melody on its blue notes while following its brutal kicks on the dons, so be prepared to break keyboards trying to pass this map. We wish you the best of luck trying to read the last kiai, which has a scrolling speed that is equal to around 392 BPM.
+
+[![](/wiki/shared/news/2018-11-18-project-loved-week-of-november-18th/taiko/mousou-express.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)
```

Comment:
was the at symbol always displaying like this?
![image](https://user-images.githubusercontent.com/14264992/48669820-b7f2b080-ead1-11e8-8802-08c2062dfdb0.png)